gp-includes/warnings.php

```diff
@@ -874,4 +874,32 @@ public function warning_unexpected_start_end_space( string $original, string $tr
 		}
 		return implode( "\n", $warnings );
 	}
+
+	/**
+	 * Checks the missing uppercase in the beginning of the translations.
+	 *
+	 * Only works with languages using latin alphabet.
```

Comment:
Did the check for the alphabet got lost due to the split into two PRs? Based on the comment we only want to enable the check for locales using latin alphabet?

Reply:
I moved the alphabet to another PR because with the new method (regex) I didn't need it. The regex looks for an initial uppercase letter that has a lowercase variant, so it doesn't depend on the Latin alphabet, because another alphabets like Cyrillic can have uppercases and lowercases.

Reply:
Ah, so we should remove the note?

Reply:
Yes, this comment is incorrect, because it applies to languages with uppercase and lowercase.

Reply:
I have removed it from the method.